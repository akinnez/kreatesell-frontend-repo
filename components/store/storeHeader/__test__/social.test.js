import Social from "../social-media";
import renderer from "react-test-renderer";

it("renders a snapshot of <Social /> with no props passed!", () => {
  const social = renderer.create(<Social />).toJSON();
  expect(social).toMatchSnapshot();
});
