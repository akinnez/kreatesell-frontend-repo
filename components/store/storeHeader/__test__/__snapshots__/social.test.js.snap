// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <Social /> with no props passed! 1`] = `
<div
  className="jsx-3632112293 social-wrapper"
>
  <p
    className="jsx-3632112293"
  >
    Share store link on
  </p>
  <div
    className="jsx-3632112293 social-icon-wrapper"
  >
    <a
      className="jsx-3632112293"
      href="#"
      target="_blank"
    >
      <svg
        fill="none"
        height="23"
        viewBox="0 0 23 23"
        width="23"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21.2031 0.953125H1.79688C1.33018 0.953125 0.953125 1.33018 0.953125 1.79688V21.2031C0.953125 21.6698 1.33018 22.0469 1.79688 22.0469H21.2031C21.6698 22.0469 22.0469 21.6698 22.0469 21.2031V1.79688C22.0469 1.33018 21.6698 0.953125 21.2031 0.953125ZM18.7668 7.10986H17.0819C15.7609 7.10986 15.5052 7.7374 15.5052 8.66025V10.6932H18.6587L18.2474 13.8757H15.5052V22.0469H12.2172V13.8783H9.46709V10.6932H12.2172V8.34648C12.2172 5.62275 13.881 4.13828 16.312 4.13828C17.4774 4.13828 18.4768 4.22529 18.7694 4.26484V7.10986H18.7668Z"
          fill="#262626"
        />
      </svg>
    </a>
    <a
      className="jsx-3632112293"
      href="#"
      target="_blank"
    >
      <svg
        fill="none"
        height="23"
        viewBox="0 0 23 23"
        width="23"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.498 7.98594C9.5627 7.98594 7.9833 9.56534 7.9833 11.5007C7.9833 13.436 9.5627 15.0154 11.498 15.0154C13.4334 15.0154 15.0128 13.436 15.0128 11.5007C15.0128 9.56534 13.4334 7.98594 11.498 7.98594ZM22.0397 11.5007C22.0397 10.0452 22.0528 8.60293 21.9711 7.1501C21.8894 5.4626 21.5044 3.96494 20.2704 2.73096C19.0338 1.49434 17.5388 1.11201 15.8513 1.03028C14.3958 0.948538 12.9535 0.961722 11.5007 0.961722C10.0452 0.961722 8.60293 0.948538 7.1501 1.03028C5.4626 1.11201 3.96494 1.49698 2.73096 2.73096C1.49434 3.96758 1.11201 5.4626 1.03028 7.1501C0.948538 8.60557 0.961722 10.0479 0.961722 11.5007C0.961722 12.9535 0.948538 14.3984 1.03028 15.8513C1.11201 17.5388 1.49698 19.0364 2.73096 20.2704C3.96758 21.507 5.4626 21.8894 7.1501 21.9711C8.60557 22.0528 10.0479 22.0397 11.5007 22.0397C12.9562 22.0397 14.3984 22.0528 15.8513 21.9711C17.5388 21.8894 19.0364 21.5044 20.2704 20.2704C21.507 19.0338 21.8894 17.5388 21.9711 15.8513C22.0555 14.3984 22.0397 12.9562 22.0397 11.5007ZM11.498 16.9086C8.50537 16.9086 6.09014 14.4934 6.09014 11.5007C6.09014 8.50801 8.50537 6.09278 11.498 6.09278C14.4907 6.09278 16.906 8.50801 16.906 11.5007C16.906 14.4934 14.4907 16.9086 11.498 16.9086ZM17.1274 7.13428C16.4287 7.13428 15.8645 6.57002 15.8645 5.87129C15.8645 5.17256 16.4287 4.6083 17.1274 4.6083C17.8262 4.6083 18.3904 5.17256 18.3904 5.87129C18.3906 6.03721 18.3581 6.20154 18.2947 6.35486C18.2313 6.50819 18.1383 6.6475 18.021 6.76482C17.9037 6.88214 17.7643 6.97517 17.611 7.03857C17.4577 7.10196 17.2934 7.13449 17.1274 7.13428Z"
          fill="#262626"
        />
      </svg>
    </a>
    <a
      className="jsx-3632112293"
      href="#"
      target="_blank"
    >
      <svg
        fill="none"
        height="23"
        viewBox="0 0 23 23"
        width="23"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21.2031 0.953125H1.79688C1.33018 0.953125 0.953125 1.33018 0.953125 1.79688V21.2031C0.953125 21.6698 1.33018 22.0469 1.79688 22.0469H21.2031C21.6698 22.0469 22.0469 21.6698 22.0469 21.2031V1.79688C22.0469 1.33018 21.6698 0.953125 21.2031 0.953125ZM7.21006 18.9276H4.08027V8.86064H7.21006V18.9276ZM5.64648 7.48428C5.2877 7.48428 4.93697 7.37788 4.63865 7.17855C4.34032 6.97922 4.10781 6.6959 3.97051 6.36443C3.83321 6.03295 3.79728 5.6682 3.86728 5.31631C3.93727 4.96442 4.11005 4.64118 4.36375 4.38748C4.61745 4.13378 4.94068 3.961 5.29258 3.89101C5.64447 3.82101 6.00922 3.85694 6.3407 3.99424C6.67217 4.13154 6.95549 4.36405 7.15482 4.66238C7.35415 4.9607 7.46055 5.31143 7.46055 5.67021C7.45791 6.67217 6.6458 7.48428 5.64648 7.48428ZM18.9276 18.9276H15.8005V14.0312C15.8005 12.8632 15.7794 11.3629 14.1736 11.3629C12.5468 11.3629 12.2963 12.6338 12.2963 13.9469V18.9276H9.17178V8.86064H12.1724V10.237H12.2146C12.6312 9.446 13.6516 8.61016 15.1756 8.61016C18.3449 8.61016 18.9276 10.6958 18.9276 13.4063V18.9276Z"
          fill="#262626"
        />
      </svg>
    </a>
    <a
      className="jsx-3632112293"
      href="#"
      target="_blank"
    >
      <svg
        fill="none"
        height="23"
        viewBox="0 0 23 23"
        width="23"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21.2031 0.953125H1.79688C1.33018 0.953125 0.953125 1.33018 0.953125 1.79688V21.2031C0.953125 21.6698 1.33018 22.0469 1.79688 22.0469H21.2031C21.6698 22.0469 22.0469 21.6698 22.0469 21.2031V1.79688C22.0469 1.33018 21.6698 0.953125 21.2031 0.953125ZM17.1769 8.5917C17.1848 8.71563 17.1848 8.84482 17.1848 8.97139C17.1848 12.8421 14.2369 17.3008 8.8501 17.3008C7.18896 17.3008 5.64912 16.8183 4.35186 15.9877C4.58916 16.0141 4.81592 16.0246 5.0585 16.0246C6.42959 16.0246 7.68994 15.5605 8.69453 14.7748C7.40781 14.7484 6.32676 13.9047 5.95762 12.7445C6.4085 12.8104 6.81455 12.8104 7.27861 12.6918C6.61608 12.5572 6.02057 12.1974 5.59327 11.6735C5.16596 11.1495 4.93322 10.4938 4.93457 9.81777V9.78086C5.32217 9.99971 5.77832 10.1342 6.25557 10.1526C5.85437 9.88526 5.52535 9.52302 5.29768 9.09803C5.07 8.67304 4.95072 8.19844 4.95039 7.71631C4.95039 7.17051 5.09277 6.67217 5.34854 6.23975C6.08392 7.14503 7.00157 7.88543 8.04185 8.41284C9.08213 8.94025 10.2217 9.24285 11.3866 9.30098C10.9727 7.31025 12.4598 5.69922 14.2475 5.69922C15.0912 5.69922 15.8506 6.05254 16.3858 6.62207C17.0477 6.49814 17.6805 6.25029 18.2447 5.91807C18.0259 6.5957 17.5671 7.16787 16.958 7.5291C17.5486 7.46582 18.1182 7.30234 18.6455 7.07295C18.2474 7.6583 17.749 8.17773 17.1769 8.5917Z"
          fill="#262626"
        />
      </svg>
    </a>
  </div>
  <div
    className="jsx-318428107 divider"
  />
  <div
    className="jsx-3632112293 copy-wrapper"
  >
    <svg
      fill="none"
      height="22"
      viewBox="0 0 20 22"
      width="20"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M14.5 0H2.5C1.4 0 0.5 0.9 0.5 2V16H2.5V2H14.5V0ZM17.5 4H6.5C5.4 4 4.5 4.9 4.5 6V20C4.5 21.1 5.4 22 6.5 22H17.5C18.6 22 19.5 21.1 19.5 20V6C19.5 4.9 18.6 4 17.5 4ZM17.5 20H6.5V6H17.5V20Z"
        fill="#323232"
      />
    </svg>
     
    <span
      className="jsx-3632112293"
    >
      Copy link
    </span>
  </div>
</div>
`;
